libpod/rootless_cni_linux.go

```diff
@@ -23,9 +23,9 @@ import (
 )
 
 var rootlessCNIInfraImage = map[string]string{
-	// Built from ../contrib/rootless-cni-infra
-	// TODO: move to Podman's official quay
-	"amd64": "ghcr.io/akihirosuda/podman-rootless-cni-infra:gd34868a13-amd64",
+	// Built from ../contrib/rootless-cni-infra.
+	// Please increase the version when uploading a new image.
+	"amd64": "quay.io/libpod/rootless-cni-infra:1-amd64",
```

Comment:
I expect the version string to be either one of:

* `VERSION` string (currently 0.1.0) in https://github.com/containers/podman/blob/4609130b0f478736fa7c63e25d4c2b7a4b20dbcd/contrib/rootless-cni-infra/rootless-cni-infra#L5
* Podman's version, i.e. "2.1.0-rc.1"
* git commit hash with "g" prefix, i.e. "gd34868a13"

Reply:
Also, it is safer to specify `@sha256` digest

Reply:
Thanks for checking.

> VERSION string (currently "0.1.0") in rootless-cni-infra script: 

That only relates to the script but not to the Containerfile. If we change the Containerfile, we had to bump a version as well.

> Podman's version, i.e. "2.1.0-rc1"

I see them as two separate things. I don't think we should version-interlock Podman and the rootless CNI image.

> git commit hash with "g" prefix, i.e. "gd34868a13"

Don't we have a chicken-egg issue as we cannot easily predict the commit hash when creating a new commit?


How about we move the VERSION variable from the script into a file `contrib/rootless-cni-infra/VERSION`? Whenever we change something, we had to touch VERSION as well. I have a preference for a simple integer but don't feel very strongly about it.

Reply:
> Don't we have a chicken-egg issue as we cannot easily predict the commit hash when creating a new commit?

This expect to have two commits: first one for `contrib/rootless-cni-infra` changes, second one for `libpod/rootless_cni_linux.go` to use the image built with the first one

> How about we move the VERSION variable from the script into a file contrib/rootless-cni-infra/VERSION?

SGTM

Reply:
> > Don't we have a chicken-egg issue as we cannot easily predict the commit hash when creating a new commit?
> 
> This expect to have two commits: first one for `contrib/rootless-cni-infra` changes, second one for `libpod/rootless_cni_linux.go` to use the image built with the first one

How could I not think about that? I evidently need a fresh coffee :see_no_evil: 

> > How about we move the VERSION variable from the script into a file contrib/rootless-cni-infra/VERSION?

Do you prefer semver or a systemd-like integer?

Reply:
> Do you prefer semver or a systemd-like integer?

Either is fine to me, can be calver, whatever 😛

Reply:
Updated, PTAL